concepts/eventing/performance evaluation/nats-jetstream/different-cloud-providers/nats-throughput-test.md

```diff
@@ -0,0 +1,111 @@
+# Throughput Evalation of NATS Jetstream on different Cloud Providers
+
+## Table of Contents
+- Test Setup
+- Test Results
+- How tests are executed
+
+## Test Setup
+* Kyma CLI version: `2.6.2`
+* Kyma: 
+  * Version: main 
+  * Production Profile
+  * JetStream with File Storage
+  * NATS JetStream Version: `2.9.0` 
+* K8s cluster:
+  * Kubernetes v1.24.6
+  * Gardener cluster [Nodes: 2(min) to 6(max)]
+  * Machine Type: `cpu: 2, memory: 16GB`
+
+* Kyma deploy command:
+    ```
+    kyma deploy --source=main --profile production ...
+    ```
+* Testing tool: [NATS Benchmarking Tool](https://docs.nats.io/using-nats/nats-tools/nats_cli/natsbench)
+<br/><br/>
+
+## Test Results
+As you see from the table all the NATS Jetstream throughput tests were similar on GCP, Azure, and AWS. 
+
+| Cloud Provider | msgs/sec (NATS Bench) | msgs | storage type | pub/sub | CPU / Memory |
+| --- | --- | --- | --- | --- | --- |
+| Azure | 4130 - 4313 | 10000 | SSD | 5/5 | 4 / 16 |
+| GCP | 2000 - 2500 | 10000 | SSD | 5/5 | 4 / 15 |
+| AWS | 3441 - 4289 | 10000 | SSD | 5/5 | 4 / 16 |
+|  |  |  |  |  |  |
+| Azure | 1630 - 2682 | 10000 | SSD | 1/5 | 4 / 16 |
+| GCP | 850  - 1320 | 10000 | SSD | 1/5 | 4 / 15 |
+| AWS | 1739 - 2071 | 10000 | SSD | 1/5 | 4 / 16 |
+|  |  |  |  |  |  |
+| GCP | 2297-2447 | 10000 | HDD | 5/5 | 4 / 16 |
```

Comment:
Where you able to provision a cluster with NATS servers backed by an HDD storage?

Reply:
yes, I specified the storage class name for the Kyma deployment. For example: 
```
kyma deploy --source=2.7.1 --profile production --value eventing.nats.nats.jetstream.fileStorage.storageClassName=managed-standard-hdd
```